examples/complete/main.tf

```diff
@@ -98,6 +98,13 @@ module "cloudfront" {
         lambda_arn = module.lambda_function.lambda_function_qualified_arn
       }
     }
+
+    function_association = {
```

Comment:
Please add similar argument for another event into `ordered_cache_behavior` block in this example

Reply:
looks like it's not possible to define both lambda_function_association and function_association in the same cache behaviour as it results in this error: `Error: error creating CloudFront Distribution: InvalidLambdaFunctionAssociation: The CloudFront function arn:aws:cloudfront::<<account_id>>:function/example-loved-cattle:LIVE with a viewer-response event type cannot be used with the Lambda function arn:aws:lambda:us-east-1:<<account_id>>:function:loved-cattle-lambda:1 with a viewer-request event type.`
So I'll define it just in the ordered cache behaviour, as there's no lambdas@edge associated there